src/query/storages/hive/hive/src/hive_parquet_block_reader.rs

```diff
@@ -191,6 +191,11 @@ impl HiveParquetBlockReader {
         }
     }
 
+    fn sync_read_column(o: &Object, offset: u64, length: u64) -> Result<Vec<u8>> {
+        let chunk = o.blocking_range_read(offset..offset + length)?;
```

Comment:
need to have it in opendal

Reply:
After https://github.com/datafuselabs/opendal/pull/1085 implemented, we can use `read_at` for better performance.